Assets/Scripts/Props/HeavyLift.cs

```diff
@@ -8,27 +8,43 @@ public class HeavyLift : PlayerDraggable
         private Vector3 _origin;
         [SerializeField] private Vector3 liftedDistanceVector;
         private BoxCollider2D _boxCollider2D;
+        private bool _isLifted;
+        private float _timeLifted;
 
         protected override void Start()
         {
             base.Start();
-            _origin = transform.position;
```

Comment:
Origin shouldn't be set when created, but when clicked. Otherwise it just shoots back to the starat